### ISSUE TITLE
Additional [[tool.poetry.source]] not taken into account

### ISSUE THREAD

**Issue by tlecarrour** [sdispater/poetry#189]
Thanks for fixing #173! installing from a private repository now works, but I cannot use 2 (or more) repositories at a time!

I have a private repository, containing only few private packages, so I would like to be able to use it with another public repository to get other packages from.

When I put the public repository first in my `pyproject.toml`, I'm able to install only public packages, not the ones from my private repository:

```
[[tool.poetry.source]]                                                          
name = "public pypi"                                                          
url = "https://pypi.python.org/simple"                                          
                                                                                
[[tool.poetry.source]]                                                          
name = "private"                                                          
url = "https://www.private.net/pypi"
```

```
$ poetry add private_package
                                
[ValueError]     
No package named "private_package"  

$ poetry add mamba
Using version ^0.9.3 for mamba
[…]
```

When I put the private repository first, it's the other way around:

```
[[tool.poetry.source]]                                                          
name = "private"                                                          
url = "https://www.private.net/pypi"

[[tool.poetry.source]]                                                          
name = "public pypi"                                                          
url = "https://pypi.python.org/simple"                                          
```

```
$ poetry add mamba
                             
[ValueError]  
No package named "mamba"

$ poetry add private_package
Using version ^0.1.0 for private_package
```

**Comment by byronrau** [comment]
Having the same issue, eta on fix?

**Comment by tlecarrour** [comment]
Fixed in develop b136c5b. Thanks!
